tarantool_test.go

```diff
@@ -2,8 +2,10 @@ package tarantool_test
 
 import (
 	"fmt"
+	"gotest.tools/assert"
```

Comment:
вы из-за трёх функций (Assert, Check, NilError) пакадж, тянущий ещё две зависимости, притащили? Нафига?

Reply:
Во-первых, это не должно быть проблемой
```
Note that Go 1.17 adds support for [module graph pruning](https://golang.org/ref/mod#graph-pruning): if your module specifies go 1.17 or higher and a consumer of your module does not use your test dependency in their own module, they will not need to download the source code or go.mod file for that dependency. (And once https://golang.org/issue/44435 is implemented, when they run go mod download it will not download the irrelevant dependency either.)
```
https://stackoverflow.com/questions/64071364/best-way-to-use-test-dependencies-in-go-but-prevent-export-them

Во-вторых, есть аналогичный PR, где применяемых функций сильно больше: https://github.com/tarantool/go-tarantool/pull/148 .

Так что поводов не подтягивать пакет не видно. Правда, тогда этот хотелось бы перевезти с `"gotest.tools/assert"` на `"github.com/stretchr/testify/require"`.

Reply:
> Правда, тогда этот PR хотелось бы перевезти с "gotest.tools/assert" на "github.com/stretchr/testify/require".

Перевез.